modules/instrument_manager/php/instrument_manager.class.inc

```diff
@@ -88,7 +88,13 @@ class Instrument_Manager extends \NDB_Menu_Filter
             );
             if ($exists <= 0) {
                 $db_config = $config->getSetting('database');
-                exec("php generate_tables_sql_and_testNames.php < $new_file");
+                exec(
+                    'php generate_tables_sql_and_testNames.php < '
+                    . escapeshellarg($new_file),
```

Comment:
This line has already been dealt with as of https://github.com/aces/Loris/pull/3612